Edit text strings for Ingest Manager Overview and Add data pages

dedemorton opened the pull request:
## Summary

Edits text strings for **Ingest Manager Overview** and **Add agent** pages to follow [EUI guidelines](https://elastic.github.io/eui/#/). 

Collaborated on these changes with @gchaps, our resident UI text guru, but she might have additional feedback after seeing this all in one place.

Please take a look at my remaining todo list. I wanted to focus on text edits only, so I'm hoping someone on the dev team will pick this up.

## Remaining TODOs

Moved to-do list to https://github.com/elastic/kibana/issues/75807.

~- [ ] Make "Gold subscription" an active link in the text "Requires a Gold subscription."~
~- [ ] Make "Datasets" an active link in the text "Go to Datasets to view your data."~
~- [ ] In the Add agent flyout, it would be helpful to add a description before the tabs so novice users will understand what an Elastic Agent is. Possibly something like: "Add Elastic Agents to your hosts to collect data and send it to the Elastic Stack."~
~- [ ] The "Start the agent" section in the standalone instructions only covers the linus/macOS flavor of the run command. Is that intentional? Why not use the same commands that are shown on the tab with the Fleet steps?~
~- [ ] In the text "Before enrolling agents, set up Fleet. Go to Fleet." Make "set up in Fleet" a link so that "Go to Fleet is not required"~



## Changed screens

Changes highlighted in yellow.

![image](https://user-images.githubusercontent.com/14206422/89968136-f31f0680-dc07-11ea-8b37-e34b71cca078.png)

<hr>

![image](https://user-images.githubusercontent.com/14206422/89968185-0df17b00-dc08-11ea-950a-45ac9c8e79d3.png)

<hr>

![image](https://user-images.githubusercontent.com/14206422/89968253-35e0de80-dc08-11ea-8465-d1415014a1ba.png)

<hr>

![image](https://user-images.githubusercontent.com/14206422/89968522-de8f3e00-dc08-11ea-8ad6-bda6a3a3653a.png)

<hr>

![image](https://user-images.githubusercontent.com/14206422/89968627-1f875280-dc09-11ea-8386-56aea83eee15.png)

<hr>

![image](https://user-images.githubusercontent.com/14206422/89968701-5eb5a380-dc09-11ea-8099-0096caad224f.png)





### Checklist

Delete any items that are not applicable to this PR.

- [x] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md)

elasticmachine commented:
Pinging @elastic/ingest-management (Team:Ingest Management)

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67642/)
* Commit: ffb047a1f3ca80eaa5f8ae29b9bd4875487af452

### Build metrics
#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/ffb047a1f3ca80eaa5f8ae29b9bd4875487af452) | diff | [baseline](https://github.com/elastic/kibana/commit/75b8a3cb710eb13c05b61c85bf345dd3e6d92cb3) |
| --- | --- | --- | --- |
| ingestManager | 1.1MB | -271.0B | 1.1MB |

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67642/","number":"67642","commit":"ffb047a1f3ca80eaa5f8ae29b9bd4875487af452"}]}
PIPELINE-->

dedemorton commented:
@elasticmachine merge upstream

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68042/)
* Commit: 7697959597cfbcf8513fc7dac20347349420ad63

### Build metrics
#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/7697959597cfbcf8513fc7dac20347349420ad63) | diff | [baseline](https://github.com/elastic/kibana/commit/ee9a8d29b1588e271486c428e423880f41fa8ee9) |
| --- | --- | --- | --- |
| ingestManager | 1.1MB | -271.0B | 1.1MB |

### History
* :green_heart: [Build #67642](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67642/) succeeded ffb047a1f3ca80eaa5f8ae29b9bd4875487af452

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67642/","number":"67642","commit":"ffb047a1f3ca80eaa5f8ae29b9bd4875487af452"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68042/","number":"68042","commit":"7697959597cfbcf8513fc7dac20347349420ad63"}]}
PIPELINE-->

dedemorton commented:
@elasticmachine merge upstream

dedemorton commented:
Ready to merge...just want @gchaps to give her blessing.

gchaps commented:
A few minor comments:

**Ingest Manager settings**

> receive the latest assets > get the latest assets

**Add agent**

> Before enrolling agents, set up Fleet.  Go to Fleet.  

Can "set up in Fleet" become the link so that "Go to Fleet is not required"?

> You can download > Download the agent 

Unless you feel using "you can" softens the instruction a bit.

> "Of the configuration file" > "elastic-agent.yml?  

We might need to be more specific here because I'm not sure what "configuration file" refers to.

> Go to Datasets to view your data 

Make Datasets a link

> You will need >  you'll 

I think the contraction sounds better, but it is your choice

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68561/)
* Commit: 634341f17528a8e7a7b7ecd4922602adebf062a0

### Build metrics
#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/634341f17528a8e7a7b7ecd4922602adebf062a0) | diff | [baseline](https://github.com/elastic/kibana/commit/df9b28a4ce39d55d76a0b1844a48e5cd1fb98245) |
| --- | --- | --- | --- |
| ingestManager | 1.1MB | -271.0B | 1.1MB |

### History
* :green_heart: [Build #68042](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68042/) succeeded 7697959597cfbcf8513fc7dac20347349420ad63
* :green_heart: [Build #67642](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67642/) succeeded ffb047a1f3ca80eaa5f8ae29b9bd4875487af452

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67642/","number":"67642","commit":"ffb047a1f3ca80eaa5f8ae29b9bd4875487af452"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68042/","number":"68042","commit":"7697959597cfbcf8513fc7dac20347349420ad63"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68561/","number":"68561","commit":"634341f17528a8e7a7b7ecd4922602adebf062a0"}]}
PIPELINE-->

dedemorton commented:
Thanks Gail. Here's what I've done:

>receive the latest assets > get the latest assets

changed

> Before enrolling agents, set up Fleet. Go to Fleet.
> Can "set up in Fleet" become the link so that "Go to Fleet is not required"?

Left as-is, but I'll add his to the to-do list. I think we might also want to change the ID, but wanted to restrict this PR to text changes only.  

> You can download > Download the agent
> Unless you feel using "you can" softens the instruction a bit.

Left as-is because because some users won't go to the download page; they will pull down the packages by using curl, apt/yu/brew when it's available, or use some automation software.

>"Of the configuration file" > "elastic-agent.yml?
>We might need to be more specific here because I'm not sure what "configuration file" refers to.

Fixed

>Go to Datasets to view your data
> Make Datasets a link

Already in the to-do list.

>You will need > you'll
>I think the contraction sounds better, but it is your choice

Fixed

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68949/)
* Commit: b176672250c2e5c8028099ffdb6c1a1b9e0ac468

### Build metrics
#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/b176672250c2e5c8028099ffdb6c1a1b9e0ac468) | diff | [baseline](https://github.com/elastic/kibana/commit/df9b28a4ce39d55d76a0b1844a48e5cd1fb98245) |
| --- | --- | --- | --- |
| ingestManager | 1.1MB | -289.0B | 1.1MB |

### History
* :green_heart: [Build #68561](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68561/) succeeded 634341f17528a8e7a7b7ecd4922602adebf062a0
* :green_heart: [Build #68042](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68042/) succeeded 7697959597cfbcf8513fc7dac20347349420ad63
* :green_heart: [Build #67642](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67642/) succeeded ffb047a1f3ca80eaa5f8ae29b9bd4875487af452

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67642/","number":"67642","commit":"ffb047a1f3ca80eaa5f8ae29b9bd4875487af452"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68042/","number":"68042","commit":"7697959597cfbcf8513fc7dac20347349420ad63"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68561/","number":"68561","commit":"634341f17528a8e7a7b7ecd4922602adebf062a0"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68949/","number":"68949","commit":"b176672250c2e5c8028099ffdb6c1a1b9e0ac468"}]}
PIPELINE-->

dedemorton commented:
@elasticmachine merge upstream

jen-huang commented:
Hi Dede, I just merged a big PR that renames agent configs to agent policies: https://github.com/elastic/kibana/pull/74914. Could you make sure the copy changes in this PR aligns with that naming too? Thanks!

dedemorton commented on x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/overview/index.tsx:
```diff
@@ -72,6 +72,7 @@ export const IngestManagerOverview: React.FunctionComponent = () => {
                 <FormattedMessage
                   id="xpack.ingestManager.overviewPageSubtitle"
                   defaultMessage="Manage Elastic Agents and their configurations in a central location."
+
```
```suggestion
```

dedemorton commented on x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/overview/index.tsx:
```diff
@@ -71,7 +71,8 @@ export const IngestManagerOverview: React.FunctionComponent = () => {
                 <FormattedMessage
                   id="xpack.ingestManager.overviewPageSubtitle"
-                  defaultMessage="Central management for Elastic Agents and agent policies."
+                  defaultMessage="Manage Elastic Agents and their configurations in a central location."
+
```
```suggestion
                  defaultMessage="Manage Elastic Agents and their configurations in a central location."
```

dedemorton replied:
```suggestion
                  defaultMessage="Manage Elastic Agents and their configurations in a central location."
```

dedemorton commented on x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/overview/index.tsx:
```diff
@@ -71,7 +71,8 @@ export const IngestManagerOverview: React.FunctionComponent = () => {
                 <FormattedMessage
                   id="xpack.ingestManager.overviewPageSubtitle"
-                  defaultMessage="Central management for Elastic Agents and agent policies."
+                  defaultMessage="Manage Elastic Agents and their configurations in a central location."
+
                 />
```
```suggestion
                  defaultMessage="Manage Elastic Agents and their configurations in a central location."
                />
```

dedemorton commented on x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/overview/index.tsx:
```diff
@@ -71,7 +71,7 @@ export const IngestManagerOverview: React.FunctionComponent = () => {
                 <FormattedMessage
                   id="xpack.ingestManager.overviewPageSubtitle"
-                  defaultMessage="Central management for Elastic Agents and agent policies."
+                  defaultMessage="Manage Elastic Agents and their configurations in a central location."
```
```suggestion
                  defaultMessage="Manage Elastic Agents and their policies in a central location."
```

dedemorton commented on x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/components/agent_enrollment_flyout/managed_instructions.tsx:
```diff
@@ -60,7 +60,7 @@ export const ManagedInstructions: React.FunctionComponent<Props> = ({ agentPolic
         <FormattedMessage
           id="xpack.ingestManager.agentEnrollment.managedDescription"
-          defaultMessage="Whether you need one agent or thousands, Fleet makes it easy to centrally manage and deploy updates to your agents. Follow the instructions below to download and enroll an Elastic Agent with Fleet."
+          defaultMessage="Enroll an Elastic Agent in Fleet to automatically deploy configuration updates and centrally manage the agent."
```
```suggestion
          defaultMessage="Enroll an Elastic Agent in Fleet to automatically deploy updates and centrally manage the agent."
```

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69213/)
* Commit: 62510d49538bf54d125e0c049bee0639cecc73c1

### Build metrics
#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/62510d49538bf54d125e0c049bee0639cecc73c1) | diff | [baseline](https://github.com/elastic/kibana/commit/9111d5096566b0d3c6659895b40f01f66450520e) |
| --- | --- | --- | --- |
| ingestManager | 1.1MB | -289.0B | 1.1MB |

### History
* :green_heart: [Build #68949](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68949/) succeeded b176672250c2e5c8028099ffdb6c1a1b9e0ac468
* :green_heart: [Build #68561](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68561/) succeeded 634341f17528a8e7a7b7ecd4922602adebf062a0
* :green_heart: [Build #68042](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68042/) succeeded 7697959597cfbcf8513fc7dac20347349420ad63
* :green_heart: [Build #67642](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67642/) succeeded ffb047a1f3ca80eaa5f8ae29b9bd4875487af452

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67642/","number":"67642","commit":"ffb047a1f3ca80eaa5f8ae29b9bd4875487af452"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68042/","number":"68042","commit":"7697959597cfbcf8513fc7dac20347349420ad63"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68561/","number":"68561","commit":"634341f17528a8e7a7b7ecd4922602adebf062a0"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68949/","number":"68949","commit":"b176672250c2e5c8028099ffdb6c1a1b9e0ac468"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69213/","number":"69213","commit":"62510d49538bf54d125e0c049bee0639cecc73c1"}]}
PIPELINE-->

dedemorton commented:
@jen-huang Thanks! Can you take a look at my changes? Sorry about the churn, but I thought I'd try using GitHub to resolve the conflicts. It's cool, but not as nice as doing them in a proper text editor. :-)

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69248/)
* Commit: 43ba78bf98f0c046433456260956d964048963f4

### Build metrics
#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/43ba78bf98f0c046433456260956d964048963f4) | diff | [baseline](https://github.com/elastic/kibana/commit/51a80c61a0e57c0257f9948cb84f0b6af12a748f) |
| --- | --- | --- | --- |
| ingestManager | 1.1MB | -289.0B | 1.1MB |

### History
* :green_heart: [Build #69213](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69213/) succeeded 62510d49538bf54d125e0c049bee0639cecc73c1
* :green_heart: [Build #68949](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68949/) succeeded b176672250c2e5c8028099ffdb6c1a1b9e0ac468
* :green_heart: [Build #68561](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68561/) succeeded 634341f17528a8e7a7b7ecd4922602adebf062a0
* :green_heart: [Build #68042](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68042/) succeeded 7697959597cfbcf8513fc7dac20347349420ad63
* :green_heart: [Build #67642](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67642/) succeeded ffb047a1f3ca80eaa5f8ae29b9bd4875487af452

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67642/","number":"67642","commit":"ffb047a1f3ca80eaa5f8ae29b9bd4875487af452"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68042/","number":"68042","commit":"7697959597cfbcf8513fc7dac20347349420ad63"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68561/","number":"68561","commit":"634341f17528a8e7a7b7ecd4922602adebf062a0"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68949/","number":"68949","commit":"b176672250c2e5c8028099ffdb6c1a1b9e0ac468"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69213/","number":"69213","commit":"62510d49538bf54d125e0c049bee0639cecc73c1"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69248/","number":"69248","commit":"43ba78bf98f0c046433456260956d964048963f4"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70106/)
* Commit: 43ba78bf98f0c046433456260956d964048963f4

### Build metrics
#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/43ba78bf98f0c046433456260956d964048963f4) | diff | [baseline](https://github.com/elastic/kibana/commit/51a80c61a0e57c0257f9948cb84f0b6af12a748f) |
| --- | --- | --- | --- |
| ingestManager | 1.1MB | -289.0B | 1.1MB |

### History
* :green_heart: [Build #69248](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69248/) succeeded 43ba78bf98f0c046433456260956d964048963f4
* :green_heart: [Build #69213](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69213/) succeeded 62510d49538bf54d125e0c049bee0639cecc73c1
* :green_heart: [Build #68949](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68949/) succeeded b176672250c2e5c8028099ffdb6c1a1b9e0ac468
* :green_heart: [Build #68561](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68561/) succeeded 634341f17528a8e7a7b7ecd4922602adebf062a0
* :green_heart: [Build #68042](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68042/) succeeded 7697959597cfbcf8513fc7dac20347349420ad63

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68042/","number":"68042","commit":"7697959597cfbcf8513fc7dac20347349420ad63"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68561/","number":"68561","commit":"634341f17528a8e7a7b7ecd4922602adebf062a0"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/68949/","number":"68949","commit":"b176672250c2e5c8028099ffdb6c1a1b9e0ac468"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69213/","number":"69213","commit":"62510d49538bf54d125e0c049bee0639cecc73c1"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69248/","number":"69248","commit":"43ba78bf98f0c046433456260956d964048963f4"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/70106/","number":"70106","commit":"43ba78bf98f0c046433456260956d964048963f4"}]}
PIPELINE-->
